pyproject.toml

```diff
@@ -35,9 +35,9 @@ pytest = "^7.2.2"
 requests_mock = "^1.9.2"
 coverage = {extras = ["toml"], version = "^7.2"}
 safety = "^2.3.5"
-mypy = "^1.1"
+mypy = "^1.6"
 typeguard = "^4.1.5"
-xdoctest = {extras = ["colors"], version = "^1.1.1"}
```

Comment:
Why ?
`xdoctest` latest version is 1.1.1 : https://github.com/Erotemic/xdoctest/releases